hoomd/md/EvaluatorPairALJ.h

```diff
@@ -1619,7 +1625,7 @@ EvaluatorPairALJ<3>::computeSimplexInteractions(const vec3<Scalar>& a,
 // Note: This method assumes that shape_i == shape_j. This should be valid for
 // all cases, and this logic will be moved up to the AnisoPotentialPair in
 // HOOMD 3.0.
-template<> std::string EvaluatorPairALJ<2>::getShapeSpec() const
+template<> inline std::string EvaluatorPairALJ<2>::getShapeSpec() const
```

Comment:
Does explicitly stating inline in a function implemented in a header actually do anything?

Reply:
Without the inline, you get multiple defined symbol errors when the header is included in multiple compilation units.

Reply:
I didn't know that was possible when the header had an include guard.

Reply:
The include guard prevents the function from being defined twice in the same compilation unit. Multiple defined symbol errors occur at the linking step when linking multiple compilation units together.